docs/data_structures/libE_specs.rst

```diff
@@ -28,7 +28,11 @@ libEnsemble is primarily customized by setting options within a ``LibeSpecs`` cl
                     Manager/Worker communications mode: ``'mpi'``, ``'local'``, or ``'tcp'``.
 
                 **nworkers** [int]:
-                    Number of worker processes in ``"local"`` or ``"tcp"``.
+                    Number of worker processes in ``"local"``, ``"threads"``, or ``"tcp"``.
```

Comment:
but not `mpi`?

Reply:
I don't believe the nworkers option has ever influenced mpi comms, only `mpirun -n N` makes a difference